_source/_posts/2022-04-07-where-okta-meets-devops.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog_post
+title: "How Can DevOps Engineers Use Okta?"
+author: edunham 
+by: advocate
+communities: [devops]
+description: "It's more than just SSO. This overview describes using Okta to access SaaS services and tooling, automate administration tasks, manage server access, and more."
+This description is excerpted in a host of places, so more detail is useful for SEO. 
```

Comment:
No idea why this turned up as a standalone line/ addition to front matter, I thought I'd added it as a comment for why I edited the description. Sorry, doesn't belong here
```suggestion

```
No idea why this was added to the front matter, it was meant as a comment for why I'd edited/embellished the description.